_includes/head/custom.html

```diff
@@ -3,3 +3,22 @@
 <link rel="icon" type="image/png" href={{ "/assets/images/icon-32.png" | relative_url }} sizes="32x32">
 <link rel="icon" type="image/png" href={{ "/assets/images/icon-96.png" | relative_url }} sizes="96x96">
 <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@300&amp;display=swap&amp;text=ROOTDataAnlysisFramework" rel="stylesheet">
+<head>
+  <link rel="manifest" href={{ "/manifest.json" | relative_url }}/>
+  <script>
+    if ("serviceWorker" in navigator) {
+  if (navigator.serviceWorker.controller) {
+    console.log("An active service worker found, no need to register");
+  } else {
+    // Register the service worker
+    navigator.serviceWorker
+      .register( {{ "/serviceWorker.js" | relative_url }}, {
```

Comment:
Almost! This gets transformed into `.register( /626/serviceWorker.js, {` - so we probably need:
```suggestion
      .register( '{{ "/serviceWorker.js" | relative_url }}', {
```